oanda_v20_platform/indicators/indicators.py

```diff
@@ -1,12 +1,13 @@
 import numpy as np
 import tulipy as ti
 import requests
-from notifier.system_logger import config_logger
+# from notifier.system_logger import config_logger
+import logging
 
 class Indicator:
     def __init__(self):
         self.indicator_base_url = 'http://api.fxhistoricaldata.com/indicators?'
-        self.global_logger = config_logger()
+        self.logger = logging.getLogger(__name__)
 
     def sma(self, data, period, ba, ohlc):
         series_arr = []
```

Comment:
I think an SMA10 on a 1h timeframe will be the same as a 60/5 *10 SMA on a 5min and pandas has a method df.rolling() which produces moving averages, then you can access any value back in time if it's in the dataframe. The function MarketData.get_daily_candles() could be converted to get a set number of 5min candles, and produce the moving average. It would also be possible and faster with numpy but more complicated to do the calcs.

Reply:
Definitely better ways to do this.  I'll start playing with that repo you told me about.